src/building-blocks/NestedFieldSet.tsx

```diff
@@ -30,6 +35,47 @@ class NestedFieldSet extends Component<IProps> {
     })(FormFieldSet);
   }
 
+  @computed
+  private get fieldSet () {
+    return getFieldSetFields(fillInFieldSet(this.props.fieldSet));
+  }
+
+  @computed
+  private get model () {
+    /*
+    This function implements the fieldConfig features
+    populateFromSearch and populateNameFromSearch
+    */
+    const { search } = this.props
+      , [firstName, lastName] = splitName(search)
+      , defaults: { [key: string]: string } = {}
+      ;
+
+    if (!search) { return defaults; }
+
+    this.fieldSet.map(fieldConfig => {
+      const {
+        field,
+        populateFromSearch,
+        populateNameFromSearch,
+      } = fieldConfig;
+
+      if (populateFromSearch) {
+        defaults[field] = search;
+      }
+
+      if (populateNameFromSearch && field === 'first_name') {
```

Comment:
Could these not be `===` but instead `endsWith`? I'm thinking of the case of, say, `plaintiff.first_name` and `plaintiff.last_name`.

The following line would have to change to something like `defaults[field] = firstName` (does `defaults` support the nested field name?)

Reply:
![image](https://user-images.githubusercontent.com/796717/51713117-f4291c80-1ffe-11e9-8856-a743586e21ea.png)